src/runtime_src/doc/toc/system_requirements.rst

```diff
@@ -16,7 +16,10 @@ Supported Xilinx® Accelerator Cards are listed in :ref:`platforms.rst`.
 XRT Software Stack for PCIe Accelerator Cards
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-XRT and OpenCL runtime require Linux kernel 3.10+ and GCC with C++14 features. 
+XRT software stack requires Linux kernel 3.10+. 
+Compiling host code with XRT OpenCL API requires GCC with C++11 features; Compiling host code with XRT native API requires GCC with C++14 features. 
```

Comment:
There really is no requirement when compiling OpenCL applications.  For example host code may be C code and `-std=c++zz` would make no sense.   The `-std=c++zz` is a requirement only if host code is C++ and is using features from c++zz.  For OpenCL we simply shouldn't mention it.

When using native C++ APIs, host code is including XRT header files that use c++14 features, so the statement to that effect is correct.